src/main/java/sg/ncl/MainController.java

```diff
@@ -290,7 +289,11 @@ public String loginSubmit(
             } else if (jwtTokenString != null || !jwtTokenString.isEmpty()) {
                 JSONObject tokenObject = new JSONObject(jwtTokenString);
                 String token = tokenObject.getString("token");
-                id = tokenObject.getString("id");
+                String id = tokenObject.getString("id");
+
+                // TODO: get user type from jwt token
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)